.github/workflows/integration-test.yml

```diff
@@ -7,7 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tutor_version: ['<18.0.0', '<19.0.0']
+        include:
```

Comment:
@BryanttV can we please explain why is this implementation necessary in the PR description? I think it's important to explain the reasoning behind this decision

-------------------------
Since I’m considering moving `tutor local do importdemocourse` to the GitHub Action, should we also pass the course ID to the environment in the step that runs the integration tests? This way, the action would handle the full demo course setup, and we can ensure that the tests always reference the correct course and prevent each plugin from setting it manually

I was able to obtain the course ID dynamically with:
```bash
COURSE_KEY=$(tutor local exec lms python manage.py lms shell -c "from openedx.core.djangoapps.content.course_overviews.models import CourseOverview; course = CourseOverview.objects.first(); print(course.id, end='')")
```

However, if the import process changes in future Open edX versions, this could break the tests. Of course, we could always introduce version-based logic to handle that, but we could also just set the course ID manually in the action (assuming the Demo course ID won’t change frequently). Starting from Tutor version 18.0.0, the ID is `course-v1:OpenedX+DemoX+DemoCourse`, so we’d handle logic for both IDs for now, and remove support for versions below 18.0.0 later

I’d like to hear your thoughts on this approach @BryanttV @mariajgrimaldi

Reply:
I really like what you propose. I think it's more appropriate to leave that responsibility to the action, than to keep it in every plugin.

Reply:
We should definitely move this to the GH action itself.

I like the proposal's automatic approach to getting the course ID. However, I'm very wary of using code from the platform without any API or interface, even from the shell. I don't think it's worth it, considering the course ID hasn't changed much over time.